src/include/duckdb/common/platform.hpp

```diff
@@ -58,6 +58,9 @@ std::string DuckDBPlatform() { // NOLINT: allow definition in header
 	os = "osx";
 #elif defined(__FreeBSD__)
 	os = "freebsd";
+#elif defined(__MVS__)
```

Comment:
Is this change necessary? You can compile with `DUCKDB_PLATFORM` to get the desired output. We prefer not to maintain a long list of platforms that we do not officially support (https://duckdb.org/docs/stable/dev/building/unofficial_and_unsupported_platforms)